src/wallet/wallet2.h

```diff
@@ -227,6 +227,11 @@ namespace tools
       BackgroundMiningNo = 2,
     };
 
+    enum ExportFormat {
```

Comment:
This file typically uses underscores instead of camel case - @moneromooo-monero change now or later?

Reply:
I don't really care, we have both I think.

Reply:
In this file in particular, there's CamelCase:

```c++
    enum RefreshType {
      RefreshFull,
      RefreshOptimizeCoinbase,
      RefreshNoCoinbase,
      RefreshDefault = RefreshOptimizeCoinbase,
    };

    enum AskPasswordType {
      AskPasswordNever = 0,
      AskPasswordOnAction = 1,
      AskPasswordToDecrypt = 2,
    };

    enum BackgroundMiningSetupType {
      BackgroundMiningMaybe = 0,
      BackgroundMiningYes = 1,
      BackgroundMiningNo = 2,
    };

    enum ExportFormat {
      Binary = 0,
      Ascii,
    };
```